gpkit/nomials.py

```diff
@@ -85,6 +85,7 @@ def __init__(self, exps=None, cs=1, varlocsandkeys=None,
                     units = cs[0]/cs[0].magnitude
                     if units.dimensionless:
                         cs = [c * ureg.dimensionless for c in cs]
+                        units = ureg.dimensionless
```

Comment:
The unit [ft/m] has a `dimensionless` property of `True`, but traps the conversion multiplier by setting the units to be the specific [dimensionless] unit, the line below coerces all the cs to that, freeing the conversion multiplier. Pints, man.